src/utils/helpers.ts

```diff
@@ -161,6 +161,16 @@ export function createGraphQlQuery(options: ICommonQueryOptions, where: string =
   if (options.orderDirection) {
     queryString += `orderDirection: ${options.orderDirection}\n`
   }
+  if (options.block) {
+    if (typeof options.block === 'object') {
+      const blockOptionJson = JSON.stringify(options.block)
+      // Remove quotes from json properties
+      const unquotedJson = blockOptionJson.replace(/"([^"]+)":/g, '$1:')
```

Comment:
why this option.block handle different from other options ?  remove quotes ?

Reply:
The block property in thegraph basically is represented as an object with two possible props - number and hash. (  https://thegraph.com/docs/graphql-api#time-travel-queries )
It's more like the where property then the other 4 props in the options that you have in the createGraphQlQuery method  (first, skip, orderBy, orderDirection). They are accepting string as a value. My approach for constructing the block prop is to convert the js object to a JSON, which actually is pretty the same as the thegraph query syntax. The only difference is that the prop shouldn't be a a quoted. So that's why I'm removing the quotes.